packages/auth/src/shared/utils.ts

```diff
@@ -104,3 +104,8 @@ export async function verifyApiSignature(
 
   return verifyApiSignatureDigest(signature as string, ctx.request, config)
 }
+
+// Intended for Date strings like "2024-12-05T15:10:09.545Z" (e.g., from new Date().toISOString())
+export function isValidDateString(date: string): boolean {
```

Comment:
small, but would be good to write a few tests for this

Reply:
got em already here https://github.com/interledger/rafiki/pull/3148/files#diff-04126f4f8bb03998a1c86d2551e9442d11fab8403c4d7aff2031088f2dc36092R149